### PR TITLE
Document metrics collection

### PR DESCRIPTION
**Pull request by dsajanice**

### CODE REVIEW

File: doc/Metrics.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+## Metrics in EdgeHub
+
+EdgeHub is intrumented to collect the following metrics:
+
+| MetricName        | Description           | Unit  |
+| ------------- |:-------------:|:-----:|
+| EdgeHubToCloudMessageLatencyMs | Time taken by EdgeHub to send a message to the cloud | Milliseconds |
```
Comment (darobs) [inline doc/Metrics.md]: Are the Latency measurements average over the reporting period, or are we reporting per message?

File: doc/Metrics.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+| EdgeHubToCloudMessageSentCount | Number of messages sent by EdgeHub to the cloud | Count per 20s |
```
Comment (darobs) [inline doc/Metrics.md]: This is count per reporting period?

File: doc/Metrics.md (hunk 1 above)
Comment (dsajanice) [reply doc/Metrics.md]: Latency is collected per message and not averaged over the period.

**Comment by dsajanice** [comment]
@darobs PTAL

**Pull request merged**